src/test/aerodromeCL.test.ts

```diff
@@ -0,0 +1,188 @@
+import { Dhedge, Pool, ethers } from "..";
+
+import { nonfungiblePositionManagerAddress } from "../config";
+import { AssetEnabled, Dapp, Network } from "../types";
+import { CONTRACT_ADDRESS, MAX_AMOUNT, TEST_POOL } from "./constants";
+import {
+  TestingRunParams,
+  beforeAfterReset,
+  setChainlinkTimeout,
+  testingHelper
+} from "./utils/testingHelper";
+import { allowanceDelta, balanceDelta } from "./utils/token";
+import INonfungiblePositionManager from "../abi/INonfungiblePositionManager.json";
+
+const testAerodromeCL = ({ wallet, network, provider }: TestingRunParams) => {
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const AERODROME_POSITION_MANGER = nonfungiblePositionManagerAddress[network][
+    Dapp.AERODROMECL
+  ]!;
+
+  const USDC = CONTRACT_ADDRESS[network].USDC;
+  const USDT = CONTRACT_ADDRESS[network].USDT;
+  const USDC_USDT_CL_GAUGE = "0xBd85D45f1636fCEB2359d9Dcf839f12b3cF5AF3F";
+  const AERO = CONTRACT_ADDRESS[network].VELO;
+
+  let dhedge: Dhedge;
+  let pool: Pool;
+  let velodromePositionManager: ethers.Contract;
+  let tokenId: string;
+  jest.setTimeout(100000);
+
+  describe(`[${network}] Aerodrome CL  tests`, () => {
+    beforeAll(async () => {
+      // top up ETH (gas)
+      await provider.send("hardhat_setBalance", [
+        wallet.address,
+        "0x100000000000000"
+      ]);
+      dhedge = new Dhedge(wallet, network);
+      pool = await dhedge.loadPool(TEST_POOL[network]);
+
+      // setChainlinkTimeout
+      await setChainlinkTimeout({ pool, provider }, 86400 * 365);
+
+      const newAssets: AssetEnabled[] = [
+        { asset: USDC, isDeposit: true },
+        { asset: USDT, isDeposit: true },
+        {
+          asset: AERODROME_POSITION_MANGER,
+          isDeposit: false
+        },
+        {
+          asset: AERO,
+          isDeposit: false
+        }
+      ];
+      await pool.changeAssets(newAssets);
```

Comment:
```suggestion
      await pool.managerLogic.changeAssets(newAssets, []);
```
I need to change it to above to get all tests passed.